### PR TITLE
Allow encoding the org name in the URL

### PR DESCRIPTION
**Pull request by ankrgyl**
This way, you can create a base url that ends with `/btorg/org name` and utilize `OPENAI_BASE_URL` to force a specific org.

### CODE REVIEW

File: packages/proxy/src/proxy.ts [hunk 1]
```diff
@@ -175,6 +175,18 @@ export async function proxyV1({
     proxyHeaders[FORMAT_HEADER],
   );
 
+  let orgName = proxyHeaders[ORG_NAME_HEADER];
+
+  const pieces = url
+    .split("/")
+    .filter((p) => p.trim() !== "")
+    .map((d) => decodeURIComponent(d));
```
Comment (manugoyal) [inline packages/proxy/src/proxy.ts:183]: should we lowercase everything too?

**Review by manugoyal — APPROVED** [review]

File: packages/proxy/src/proxy.ts (hunk 1 above)
Comment (ankrgyl) [reply packages/proxy/src/proxy.ts:183]: org names are case sensitive?

Comment (manugoyal) [reply packages/proxy/src/proxy.ts:183]: True. I was thinking about the `btorg` part, maybe we just lowercase that comparison.

Comment (ankrgyl) [reply packages/proxy/src/proxy.ts:183]: done

**Pull request merged**